pdr_backend/accuracy/utils/get_start_end_params.py

```diff
@@ -0,0 +1,19 @@
+from datetime import datetime, timedelta
+from typing import Tuple
+
+
+def get_start_end_params(contract_timeframe: str) -> Tuple[int, int]:
```

Comment:
This is a weird module name, and function name, for what it does. 

And, you shouldn't need a new module.

Reply:
renamed as "calculate_timeframe_timestamps"